src/common/logo.c

```diff
@@ -249,6 +269,37 @@ static void loadGarudaLogo(FFlogo* logo, bool doColor)
     sprintf(logo->chars[19], FASTFETCH_TEXT_MODIFIER_BOLT"%s                                              "FASTFETCH_TEXT_MODIFIER_RESET, color);
 }
 
+static void loadLogoFromFile(FFlogo* logo, bool doColor, char* pathToFile)
+{
+    strcpy(logo->name, "amogus");
+
+    const char* color = doColor ? "\033[91m" : "";
+    strcpy(logo->color, "\033[91m");
+
+    FILE* fp = fopen(pathToFile, "r");
+    char buff[255];
+    char* normalizedBuff;
+    int len;
+
+    int logoWidth = 0, logoHeight = 0;
+    while(!feof(fp)) {
+        fgets(buff, sizeof(buff), fp);
+        normalizedBuff = strtok(buff, "\n");
+        
+        sprintf(logo->chars[logoHeight++],   FASTFETCH_TEXT_MODIFIER_BOLT"%s%s"FASTFETCH_TEXT_MODIFIER_RESET, color, normalizedBuff);
+
+        len = utf8_strlen(normalizedBuff);
+        logoWidth = len > logoWidth
+            ? len
+            : logoWidth;
+    }
```

Comment:
Wouldn't it be simpler to use `getline`?

Reply:
I don't see how using getline would make it simpler, getline gets the same data as fgets, I still have to strip the newline character from the read data

Reply:
`getline` returns the number of bytes read, so removing the 'ņ' is faster.